modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-vsphere_{context}"]
+= vSphere automatic migration
+
+=== New installations of {product-title}
+For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+
+=== Upgrading {product-title}
+When upgrading from {product-title} 4.12, or earlier, to 4.13, or later, automatic CSI migration for vSphere only occurs if you opt in.
+
+Note the following consequences when opting in to migration:
+
+* Enabling migration cannot be undone.
+
+* Migration can take awhile to complete depending on how many nodes are on the cluster.
+
+* Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
+
+* In some cases, persistent volumes (PVs) created using the vSphere in-tree driver do not attach to nodes after migration.  
```

Comment:
This statement is vague. I think we should provide some links to trackers or to a KB that provides more details. @gnufied @jsafrane WDYT?

Reply:
I have pinged @sferich888 about whether these known issues should be a KCS or documented inline.

Reply:
happy to create KCS for this; however there are 3 known issues (do we expect to find / identify more)? Do all of the 'known issues' have remediations?

Reply:
The remediations are similar and hence one KCS imo make sense.

Reply:
https://access.redhat.com/node/7011683 is a 'header' (KCS - not really); that links to 3 uniquely defined KCS (using tags, so we can add/remove things on the fly-quickly without updates to docs.) https://access.redhat.com/solutions/7011684, https://access.redhat.com/solutions/7011685 and https://access.redhat.com/solutions/7011681.

Reply:
@gnufied let me know if we can close https://issues.redhat.com/browse/PLMCORE-5960

Reply:
Done.